browser-test/src/support/admin_programs.ts

```diff
@@ -1188,4 +1188,8 @@ export class AdminPrograms {
     await this.page.getByText('Back').click()
     await this.page.waitForLoadState()
   }
+
+  async isPaginationVisible(): Promise<boolean> {
+    return this.page.locator('.usa-pagination').isVisible()
```

Comment:
(question) Are we likely to ever need two pagers on the screen?

Reply:
Probably not likely, but I'll add a more specific locator just in case.

Reply:
I'd recommend using `page.getByRole`

```typescript
await page.getByRole('navigation', { name: 'whatever-the-name' })`
```

https://playwright.dev/docs/api/class-page#page-get-by-role

Reply:
Assuming the pager has an aria role/label on it. I don't recall what the new one has on it.

Reply:
Done